src/App.tsx

```diff
@@ -1,8 +1,36 @@
+import {
+  Routes, Route, Link, Navigate,
+} from 'react-router-dom';
 import './App.scss';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFoundPage } from './components/NotFoundpage';
 
 const App = () => (
   <div className="App">
-    <h1>People table</h1>
+    <header>
+      <nav className="navigation">
+        <Link
+          className="navigation__link"
+          to="/"
+        >
+          Home
+        </Link>
+        <Link
+          className="navigation__link"
+          to="/people"
+        >
+          People
+        </Link>
+      </nav>
+    </header>
+
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="/people" element={<PeoplePage />} />
+      <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
`replace`?
here is why – https://reach.tech/router/api/navigate
also – this one
![Screen Shot 2022-08-05 at 09 31 42](https://user-images.githubusercontent.com/75496546/183015874-4febb8dc-7972-4e73-a320-65a2a52a4aca.png)